src/styles/_variables.scss

```diff
@@ -0,0 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
```

Comment:
Why _variables? Why the underscore? They are not variables btw, they are constants. I suggest removing the underscore and separating it into color.scss, and constant.scss (does not matter if singular or plural)